pallets/amm/src/weights.rs

```diff
@@ -36,79 +37,78 @@
 // pallets/amm/src/weights.rs
 // --template=.maintain/pallet-weight-template.hbs
 
+
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::{
-	traits::Get,
-	weights::{constants::RocksDbWeight, Weight},
-};
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
 use sp_std::marker::PhantomData;
 
 /// Weight functions needed for amm.
 pub trait WeightInfo {
-	fn create_pool() -> Weight;
```

Comment:
sidenote, these generated files should be formatted by rustfmt before committed

Reply:
yes. i noticed that too.  There is hbs template for this file. Probably would be enough to reformat the template. There is also a bug in this template ( but was fixed on some branches ). I will add separate issue for easy tracking.